This patch makes the pyconfig.h header file 32/64 bit friendly.
The patch also modifies cflags.
As it is Solaris-specific, it is not suitable for upstream.

--- Python-2.7.15/configure.ac.orig	2018-11-13 00:42:04.434512787 -0800
+++ Python-2.7.15/configure.ac	2018-11-13 00:45:55.453873428 -0800
@@ -5,7 +5,7 @@
 # Set VERSION so we only need to edit in one place (i.e., here)
 m4_define(PYTHON_VERSION, 2.7)
 
-AC_PREREQ(2.65)
+AC_PREREQ(2.68)
 
 AC_REVISION($Revision$)
 AC_INIT(python, PYTHON_VERSION, https://bugs.python.org/)
@@ -1106,7 +1106,7 @@
 	;;
 
     *)
-	OPT="-O"
+	OPT=""
 	;;
     esac
 fi
@@ -2295,12 +2295,6 @@
 if test -z "$CCSHARED"
 then
 	case $ac_sys_system/$ac_sys_release in
-	SunOS*) if test "$GCC" = yes;
-		then CCSHARED="-fPIC";
-		elif test `uname -p` = sparc;
-		then CCSHARED="-xcode=pic32";
-		else CCSHARED="-Kpic";
-		fi;;
 	hp*|HP*) if test "$GCC" = yes;
 		 then CCSHARED="-fPIC";
 		 else CCSHARED="+z";
